build-tools/packages/build-cli/src/commands/generate/manifest.ts

```diff
@@ -0,0 +1,226 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fetch from "node-fetch";
+import { promises as fsPromises } from "fs";
+import { Flags } from "@oclif/core";
+import { Logger } from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+const PACKAGE_NAME = "@fluidframework/container-runtime";
+const GITHUB_RELEASE_URL = "https://api.github.com/repos/microsoft/FluidFramework/releases";
+
+interface IBuildDetails {
+	definition: { name: string };
+	status: string;
+	result: string;
+	sourceBranch: string;
+	finishTime: string;
+	buildNumber: string;
+}
+
+export class GenerateManifestFile extends BaseCommand<typeof GenerateManifestFile> {
+	static description = `As all builds are published to build feed, this command picks the latest successful build against the provided branch name and generates dev manifest file.`;
+
+	static flags = {
+		organization: Flags.string({
+			description: "Organization name",
+			char: "o",
+			required: true,
+		}),
+		project: Flags.string({
+			description: "Project name",
+			char: "p",
+			required: true,
+		}),
```

Comment:
Do you need these separately? If not, I would just have a single flag for the repo.